plugins/auth-backend/src/providers/providers.ts

```diff
@@ -52,3 +53,26 @@ export const providers = Object.freeze({
   onelogin,
   saml,
 });
+
+/**
+ * All auth provider factories that are installed by default.
+ *
+ * @public
+ */
+export const defaultAuthProviderFactories: {
+  [providerId: string]: AuthProviderFactory;
```

Comment:
Just a thought here - is it good or bad that this one shades over what the actual keys of the underlying type is? It's helpful to see them, in a sense.